### DISCUSSION TITLE
Devtools blocked

### DISCUSSION THREAD

**Discussion by RacoonyCoon** [3kh0/ext-remover#1352] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello today I found out they blocked devtools we use iboss. They also recently removed a lot of extensions like ublock origin and other adblockers so javascript currently is not useable I am on 126 is there anything I can do to be able to execute JavaScript and disable extensions like devtools?

**Comment by schoolexploitkid** [comment]
rigtools: https://github.com/3kh0/ext-remover#:~:text=Back%20to%20top-,Rigtools,-Rigtools%20is%20an

**Reply by RacoonyCoon** [reply]
What was the point of this?

**Reply by OPcoolman** [reply]
just use like an extension hanger that works for like 5 minutes and use rigtools

**Reply by RacoonyCoon** [reply]
The extension hanger doesn't really work for me even if it worked its blocked by administrator not by anything else because I have 127.0.0.1 and localhost active with a VPN so I already have everything unblocked.

**Reply by Exploiter-org** [reply]
What's localhost

**Comment by Exploiter-org** [comment]
Im on version 130 so devtools don't work for me anymore

**Comment by WildcatGamesUnblocked** [comment]
well rn the devtools are not working the websocket is disconnected

**Reply by Exploiter-org** [reply]
For me it dosent work because im on v130

**Reply by OPcoolman** [reply]
If you used it to disable extensions on a older version you can still do it

**Reply by Exploiter-org** [reply]
I used to disable extensions on v128

**Reply by OPcoolman** [reply]
Imma show you when I get home

**Reply by Exploiter-org** [reply]
Ok thx

**Reply by Exploiter-org** [reply]
How much mins

**Reply by Exploiter-org** [reply]
Like I did the same steps i did in v128

**Reply by OPcoolman** [reply]
Idk what mins lol

**Reply by Exploiter-org** [reply]
Well could u tell me now?

**Reply by Exploiter-org** [reply]
I'll just wait till u get home

**Reply by OPcoolman** [reply]
is that good

**Reply by Exploiter-org** [reply]
It's not in my history bc I reseted my Chromebook

**Reply by OPcoolman** [reply]
Downgrade

**Reply by Exploiter-org** [reply]
I do have it bookmarked tho

**Reply by Exploiter-org** [reply]
How do I downgrade

**Reply by Exploiter-org** [reply]
Pls tell me I don't need a usb

**Reply by Exploiter-org** [reply]
I just got a old computer that's on v119 lol

**Reply by Charlieis1nsan3** [reply]
bruh if your on v119 you have a wide variety of exploits you can use, check your kernver also. 
and also Wildcatgames,if it keeps disconnecting, its either because you are A. doing it wrong, B. On v130 or up, or C. your school is using DNS blocking, try it at home. (if you can)

**Reply by OPcoolman** [reply]
if you used rigtools to disable extension on an older version you can still do it on the new one

**Reply by OPcoolman** [reply]
> Pls tell me I don't need a usb

yea you need a usb lol

**Reply by Exploiter-org** [reply]
I downgraded to 125

**Reply by Exploiter-org** [reply]
Any exploits besides rigtools

**Comment by S-PScripts** [comment, marked as answer]
rh
